akka-typed-tests/src/test/scala/akka/typed/scaladsl/ImmutableOrUnhandledSpec.scala

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.typed
+package scaladsl
+
+import akka.typed.testkit.{ EffectfulActorContext, TestKitSettings }
+import akka.typed.testkit.scaladsl.TestProbe
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import scala.concurrent.duration.DurationInt
+
+@RunWith(classOf[JUnitRunner])
+final class ImmutableOrUnhandledSpec extends TypedSpec {
+
+  final object `An Actor.immutableOrIgnore behavior (native)`
+    extends Tests
+    with NativeSystem
+
+  final object `An Actor.immutableOrIgnore behavior (adapted)`
```

Comment:
`Actor.immutableOrUnhandled` for both these two

Reply:
Thanks!